flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/data/AbstractTestFlinkAvroReaderWriter.java

```diff
@@ -156,7 +154,7 @@ private Record recordNumType(
 
   @Test
   public void testNumericTypes() throws IOException {
-
+    
```

Comment:
removing that line in your earlier commit was fine imo, but this is adding a tab ;)